[NETBEANS-54] Module Review languages.diff

bsungur opened the pull request:
fixes #123

Add ASF header to DiffExample.diff

bsungur committed:
Revert "[NETBEANS-54] Module Review languages.diff"

This reverts commit a0a80b0a7dda454c3270d9cdb86bbc346bb58552.

bsungur committed:
[NETBEANS-54] Module Review languages.diff

fixes #123

Add ASF header to DiffExample.diff

emilianbold reviewed (changes requested):
This can't have a license header as it's user visible in Options | Fonts and Color | Syntax / Diff File.

Please exclude it from nbbuild/build.xml ("rat" task)

bsungur committed:
[NETBEANS-54] Module Review languages.diff

Add rat exclude for DiffExample.diff since a license header would be visible to users in Fonts and Color settings.

bsungur commented:
Reverted commit that added license header to DiffExample.diff file. Added exclude in rat task.

emilianbold committed:
Merge branch 'master' into 123-languages.diff
